src/pages/Experiments/Experiment/Drawer/GenericDrawer/GenericDrawerContainer.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
@fberanizo @LuanEdCosta @SchafferCPqD 

Acredito que na nova versão do react-scripts, não seja necessário o uso do import do React.

Continuamos a mantê-lo ou removemos?

----

Outro detalhe, quando formos renomear um arquivo, primeiro renomeamos e commitamos, depois realizamos qualquer alteração que seja necessária... isso é para manter a rastreabilidade do arquivo. Se renomeamos e alteramos no mesmo commit, o git se perde e entende que um novo arquivo foi criado.

Reply:
Sim o CRA 4.x.x já permite tirar o `import React` por causa da nova transformação do JSX. Por mim tanto faz tirar ou deixar mas talvez dê pra fazer isso com script. https://pt-br.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Esse detalhe de renomear arquivo é verdade, mas esqueço de fazer algumas vezes :sob:.

Reply:
https://pt-br.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports

vc fala isso?

Reply:
Isso mesmo, um `codemod`. Nunca usei nada do tipo, mas vale a pena testar em uma branch, só que vai dar muito conflito, o ideal é ser o último PR da sprint.